.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] <component>: <headline>"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Always mention the tag / hash on which this was tested AND how often it was reproduced (for example - 1/10 tries).
+
+**To Reproduce**
```

Comment:
Isn't it more appropriate to use a heading here?
```suggestion
## To Reproduce
```

Note that the **Describe the bug** doesn't need to be a heading IMO.